plugins/modules/openafs_principal.py

```diff
@@ -689,6 +689,12 @@ class OracleMITKerberosAdmin(MITKerberosAdmin):
     kadm5_acl = '/var/kerberos/krb5kdc/kadm5.acl'
 
 
+class OracleMITKerberosAdmin(MITKerberosAdmin):
```

Comment:
I think the class name should be opensuseMITKerberosAdmin.  (copy and paste error?)

Reply:
I'll merge the pr, then fix the class name.